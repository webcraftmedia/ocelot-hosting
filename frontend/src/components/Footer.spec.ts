
import { renderSuspended } from '@nuxt/test-utils/runtime'
import Component from './Footer.vue'
import { describe, expect, it } from 'vitest'

describe('Footer', () => {
  it('renders', async () => {
    const html = await renderSuspended(Component, { route: '/' })
    expect(html).toMatchSnapshot()
  })
})