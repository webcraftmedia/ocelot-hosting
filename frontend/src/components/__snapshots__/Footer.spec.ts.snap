// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer > renders 1`] = `
"<div id="test-wrapper">
  <footer class="bg-white rounded-lg shadow-sm m-4 dark:bg-gray-800">
    <div class="w-full mx-auto max-w-screen-xl p-4 md:flex md:items-center md:justify-between"><span class="text-sm text-gray-500 sm:text-center dark:text-gray-400">Â© 2025 <a href="https://www.webcraft-media.de/" class="hover:underline">Webcraft-Media</a>. All Rights Reserved.</span>
      <ul class="flex flex-wrap items-center mt-3 text-sm font-medium text-gray-500 dark:text-gray-400 sm:mt-0">
        <li><a href="https://www.webcraft-media.de#!impressum" class="hover:underline me-4 md:me-6">Imprint</a></li>
        <li><a href="https://www.webcraft-media.de#!datenschutz" class="hover:underline me-4 md:me-6">Privacy Policy</a></li>
        <li><a href="#" class="hover:underline me-4 md:me-6">Licensing</a></li>
        <li><a href="#" class="hover:underline">Contact</a></li>
      </ul>
    </div>
  </footer>
</div>"
`;
